### PR TITLE
Riders Module

### PR DESCRIPTION
**Pull request by jimbrig**
see #52

### CODE REVIEW

**Comment by jimbrig** [comment]
@merlinoa ready for review; just added a simple DT

image

File: shiny_app/R/5_riders_module.R [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    req(riders_prep())
+
+    out <- riders_prep()
+
+    n_row <- nrow(out)
+    n_col <- ncol(out)
+    cols <- snakecase::to_title_case(colnames(out))
+    # esc_cols <- c()
+    id <- session$ns("riders_table")
```
Comment (merlinoa) [inline shiny_app/R/5_riders_module.R:69]: what is this for?  Remove it if it is not used.

File: shiny_app/R/5_riders_module.R [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          hr()
+        )
+      )
+    )
+  )
+}
+
+riders_module <- function(input, output, session, vendor_info) {
+
+  ns <- session$ns
```
Comment (merlinoa) [inline shiny_app/R/5_riders_module.R:26]: It looks like this is not being used.  Remove if if it is not being used.

File: shiny_app/R/5_riders_module.R [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+  })
+
+  output$riders_table <- DT::renderDT({
+
+    req(riders_prep())
+
+    out <- riders_prep()
+
+    n_row <- nrow(out)
```
Comment (merlinoa) [inline shiny_app/R/5_riders_module.R:65]: These `n_row` and `n_col` variables are also not used.  Remove everything that is not used.

**Review by merlinoa — CHANGES_REQUESTED** [review]
Minor changes to clean up unused code.  Go ahead and merge after code cleanup per my comments.

**Pull request merged**